### DISCUSSION TITLE
Can't open control centre

### DISCUSSION THREAD

**Discussion by winterec** [regolith-linux/regolith-desktop#795] [General]
I haven't been able to open my control centre for over a month per this issue: https://github.com/regolith-linux/regolith-desktop/issues/738

I don't know how to repair this on my own and it's getting to the point that I may have to abandon Regolith after years of daily use because it's effecting my day-to-day productivity.

Could we please get some eyes onto the issue? Or if anyone has experienced it could you please share any workarounds?

**Comment by dmikushin** [comment]
Hey @winterec I think you are hit by the same issue as presented here: https://github.com/regolith-linux/regolith-desktop/discussions/794 could you please check? Don't give up, we should get it fixed, with some patience.

**Comment by winterec** [comment]
@dmikushin Thank you, installing `regolith-i3-control-center-gnome` has resolved this for me.

**Comment by kgilmer** [comment]
Please see here for further details regarding this problem: https://github.com/orgs/regolith-linux/discussions/794#discussioncomment-4468819

Summary:

```
In summary, for i3 keybindings that launch upstream settings (gnome-control-center), install the regolith-i3-control-center-gnome package, and for regolith settings, install regolith-i3-control-center-regolith.
```
